### PR TITLE
Rad branch

### PR DESCRIPTION
**Pull request by chihaby**
edded <p>

### CODE REVIEW

**Review by Larrylapnguyen — CHANGES_REQUESTED** [review]
Remove second p tag

File: Larry/p-index.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<title>Document</title>
+</head>
+
+<body>
+	<h1>Larry</h1>
+	<p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Veniam, sapiente aliquam tenetur modi quo officiis autem!
+		Voluptatibus deleniti eius quas.</p>
+	<p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Illo, rerum!</p>
+	<p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Veniam, sapiente aliquam tenetur modi quo officiis autem!
```
Comment (Larrylapnguyen) [inline Larry/p-index.html:16]: `<h1>Hello</h1>`

**Review by Larrylapnguyen — COMMENTED** [review]
Test

File: Larry/p-index.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<title>Document</title>
+</head>
+
+<body>
+	<h1>Larry</h1>
```
Comment (chihaby) [inline Larry/p-index.html:12]: add h2 tag

**Review by chihaby — COMMENTED** [review]
add tag

**Review by Larrylapnguyen — APPROVED** [review]

**Pull request merged**